plugins/com.google.cloud.tools.eclipse.appengine.libraries/schema/com.google.cloud.tools.eclipse.appengine.libraries.exsd

```diff
@@ -0,0 +1,194 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="com.google.cloud.tools.eclipse.appengine.libraries" xmlns="http://www.w3.org/2001/XMLSchema">
```

Comment:
Looking at the Eclipse JDT schemas the `targetNamespace` seems to be the bundle's symbolic name, [example](https://github.com/eclipse/eclipse.jdt.core/blob/947633cd3e536904a3c62f9fa98185b52f85fb30/org.eclipse.jdt.core/schema/classpathContainerInitializer.exsd#L3)